spec/api/v2/ccr_claim_spec.rb

```diff
@@ -20,7 +20,7 @@
   end
 end
 
-describe API::V2::CCRClaim do
+RSpec.describe API::V2::CCRClaim, feature: :injection do
```

Comment:
❓ Is this deliberate to be `feature:` instead of `type:` ?

Reply:
yeah, for me the type would be `:entity`. This was just a tag to enable me to do this

`rspec --tag feature:injection`
`rspec --tag type:adapter --tag feature:injection`